docs/blog/2018-02-26-documentation-project/index.md

```diff
@@ -0,0 +1,61 @@
+
+## How much people like the docs :D
+
+In over 60 interviews with Gatsby users, I asked “What do you like about Gatsby?” Documentation comes out as one of the clear winners, as you can see in this list of the top ten things interviewees like about Gatsby, courtesy of Unbird.com analytics:
+
+![Unbird.com analytics screenshot](unbird-gatsby-likes.png)
+
+To dive a little deeper, here are some quotes about how much people like the docs and tutorials:
+
+> “Very advanced docs & tutorials for this stage in a project.”
+> “Documentation is wonderful.”
+> “For me personally, the most important thing to help me reach my goals with Gatsby is good documentation with examples - which is, in my opinion, already the case compared to other projects.”
+> “Tutorials are very beginner friendly and great for someone who wants to become a better developer.”
+> “The tutorials are very easy to understand and go through great lengths to explain every line of code necessary.”
+
+So the bottom line: nearly everyone we’ve interviewed loves the Gatsby development experience and the docs, guides, and tutorial. 
+
+So why fix what isn’t broken?
+
+## Enhancing the Tutorials
+
+Just because the tutorials and docs are awesome and quite advanced doesn’t mean they can’t still grow! And we want them to be the absolute _best_ they can be.
+
+Many of you have given me valuable feedback during UX research interviews on ways to enhance the tutorials. Here is a brief overview of the issues (see Issue #4175 on Github to follow this and contribute):
+
+Gatsby is beginner-friendly; however, _true_ beginners to programming need an intro to basic tools like the command line, code editors, and browser consoles.
+Upon first visit to Gatsbyjs.org, most visitors click “Get Started.” This is the perfect solution for them if they prefer to jump into coding without reading much. If they prefer step-by-step instructions, it takes some looking around before they see the tutorial tab across the top.
+Installing new starters for every part of tutorial feels like a distraction to some beginners.
+"Part 1" is not a descriptive title for a tutorial if someone wants to preview or remember what was covered.
+Almost everyone is new to GraphQL and gets overwhelmed by Part 4.
+
+## Documentation project beginnings
+
+After outlining the issues above and attempting to solve them myself, I realized the pooled efforts of the community would be incredibly valuable in creating solutions to these issues. So, I created the Documentation Project in Github to plan and track documentation issues.
+
+The [Documentation Project’s](https://github.com/gatsbyjs/gatsby/projects/3) goal is to make Gatsby documentation clear, organized, and useful. Issues in the project come from the community and/or interviews with community members. This project also provides a centralized way to plan, manage, and track our progress so that we maintain and enhance the docs and tutorials.
```

Comment:
Good question! This project is new and ought to last for quite a while (i.e. probably the rest of the year at least, if not years), and so this concern would matter down the road. Any recommendations for how to handle that?

Reply:
Even if the project is completed, wouldn't GitHub keep a link to it (e.g. closed issues are still available at the same links as when they're open)?

Reply:
it almost certainly would — though "project" is a misnomer here because we'll be using this board forever more. We're using it more like a Trello board.